GAE/src/com/gallatinsystems/survey/domain/Question.java

```diff
@@ -16,16 +16,11 @@
 
 package com.gallatinsystems.survey.domain;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.TreeMap;
+import com.gallatinsystems.framework.domain.BaseDomain;
 
 import javax.jdo.annotations.NotPersistent;
 import javax.jdo.annotations.PersistenceCapable;
-
-import com.gallatinsystems.framework.domain.BaseDomain;
+import java.util.*;
```

Comment:
This is a default setting on IntelliJ IDEA, after 5 imports of the same package, you get `.*` which not desirable.